### PR TITLE
Bugfix/fix migration cli bugs

### PR DESCRIPTION
**Pull request by SerialVelocity**
Fix migration cli bugs:
  - Decrypt encrypted values
  - Use `--inline-config` if `-mc is not present`

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/1171)

### CODE REVIEW

File: atlasdb-config/build.gradle [hunk 1]
```diff
@@ -13,9 +13,10 @@ dependencies {
     // versions below 8.10.0 have a bug where POST requests must have a body
     compile group: 'com.netflix.feign', name: 'feign-okhttp'
     compile group: 'javax.validation', name: 'validation-api'
+    compile group: 'com.palantir.config.crypto', name: 'encrypted-config-value-module'
 
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
-    compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:' + libVersions.jackson
+    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml'
```
Comment (jeremyk-91) [inline atlasdb-config/build.gradle]: Do we need to specify `version: libVersions.jackson`? I'm aware you've done some work on enforcing specific versions of dependencies, which might mean that this is not necessary though.

**Comment by jeremyk-91** [comment]
Looks good to me - just check the version thing is okay and this should be good to go.

**Comment by tpetracca** [comment]
@j-baker can you confirm that this is how you'd actually like to interact with the kvs migration cli through the dropwizard bundle?  This basically says that you still have to specify a one-off config file for the old kvs, and that the new kvs will be read directly from your <service>.yml

File: atlasdb-config/build.gradle (hunk 1 above)
Comment (gsheasby) [inline atlasdb-config/build.gradle]: Not necessary, as there's a matching line in [versions.props](https://github.com/palantir/atlasdb/blob/bugfix/fix-migration-cli-bugs/versions.props#L5).

**Comment by j-baker** [comment]
Yes

**Comment by jboreiko** [comment]
Would be great if we could document this.  Otherwise people will have no way of knowing how to use the migration command in the dropwizard bundle.  That way the all products can just direct to our docs.

**Pull request merged**